### ISSUE TITLE
non-representatives to not vote on behalf of a project

### ISSUE THREAD

**Issue by koen84** [graphprotocol/mission-control-curator#143]
As i understand non-representative project creators get to make challenges / votes.  The project and the creator might not have the same opinion, which might lead to problems and frictions in any of the involved communities.  

Imagine "project X" challenging / voting on the fate of project Y, but the challenge / vote is not coming from someone on behalf of project X.  This creates potential perception issues, that could find backlash in project X&Y.  (And maybe also in everest as platform builder.)

**Comment by Web3Slimchance** [comment]
I am thinking the same. We get situations where "ProjectX challenged to remove ProjectY" This doesn't look very professional. Especially if the projects are competitors etc. 

Tying into my issue https://github.com/graphprotocol/mission-control-curator/issues/14 ; I would much rather have individuals with voting power dependent on their contribution. Or there could be two types of accounts -  Dev accounts and contributor accounts ?

**Comment by itsjerryokolo** [comment]
I think for Everest to work, we need clearly defined rules. Challenges and Votes shouldn't be based on opinions.
As long as the rules are being followed non-representatives should be able to participate.

**Comment by itsjerryokolo** [comment]
> 
> 
> I am thinking the same. We get situations where "ProjectX challenged to remove ProjectY" This doesn't look very professional. Especially if the projects are competitors etc. Tying into my issue #14 ; I would much rather have individuals with voting power dependent on their contribution. Or there could be two types of accounts - Dev accounts and contributor accounts ?

I agree with the idea of accounts. I think if we have rules to determine when a project should or should not be removed then everyone is on the same page

**Comment by itsjerryokolo** [comment]
Potentially, these accounts could determine the limits of what anyone can do.
In a way it still goes back to clearly defined rules.

**Comment by koen84** [comment]
> I think for Everest to work, we need clearly defined rules. Challenges and Votes shouldn't be based on opinions.
> As long as the rules are being followed non-representatives should be able to participate.

There'd still be perception, that's a powerful factor.  And we need to acknowledge human psychology, nobody is a robot.  (You might up challenging a challenger.)

**Comment by kimorq** [comment]
> As i understand non-representative project creators get to make challenges / votes. The project and the creator might not have the same opinion, which might lead to problems and frictions in any of the involved communities.
> 
> Imagine "project X" challenging / voting on the fate of project Y, but the challenge / vote is not coming from someone on behalf of project X. This creates potential perception issues, that could find backlash in project X&Y. (And maybe also in everest as platform builder.)

Came to post the same idea. I was being hesitant on voting on some challenges as I am not the owner of the projects I created and this doesn't feel right.

**Comment by yanivtal** [comment]
Great discussion. I think the issue here is we designed challenges to come from projects since reputation is based on how long the project has been on the registry. This was done to simplify the design. I like exploring the design space of having different roles and seeing how else we can calculate reputation. With the current design, a challenge has to be shown as coming from a project which I agree, is a little strange especially if the owner of the listing is not actually from the project.

@itsjerryokolo your comment is why Everest has a [Charter](http://everest.link/charter/). The idea of the charter is to make the rules for how to vote as objective as possible. If there are grey areas where people have to vote using subjective opinions, that should get discussed and the charter should be amended so the rules of the registry are well specified.
